src/menu/settings_menu_template.js

```diff
@@ -12,6 +12,15 @@ export const settingsMenu = {
         item.checked = autoHideMenuPref;
         window.setAutoHideMenuBar(autoHideMenuPref);
       }
+    },
+    {
+      label: "Start In Tray",
+      type: "checkbox",
+      click: (item, window) => {
```

Comment:
Since you're not doing anything related to `window` within the function you can remove it from the parameters, changing it to just `(item)`.